.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,77 @@
+        - [ ] Give a heads-up on the channel `#staging-headsup` that you are starting the deployment.
+        - [ ] Create a draft pull request on dd-go upgrading dd-trace-go to the release candidate tag:
+          ```console
+          dd-go/$ git checkout prod
+          dd-go/$ git pull
+          dd-go/$ git checkout -b dd-trace-go-v1.2.3
+          dd-go/$ go get -u -v gopkg.in/DataDog/dd-trace-go.v1@v1.2.3-rc.4
+          dd-go/$ go mod tidy
+          dd-go/$ git add go.mod go.sum
+          dd-go/$ git commit -m '[go.mod] upgrade dd-trace-go to v1.2.3-rc.4'
+          dd-go/$ git push -u origin dd-trace-go-v1.2.3
+          ```
+        - [ ] Deploy some services to staging:
+          ```console
+          dd-go/$ env GITLAB_TOKEN=MY_TOKEN to-staging trace-sampler \
+                                                 trace-fetcher \
+                                                 trace-firehose-writer \
+                                                 trace-stats-aggregator \
+                                                 trace-stats-extractor \
+                                                 trace-stats-query \
+                                                 fleet-config-delivery \
+                                                 runtime-security-rule-checker \
+                                                 appsec \
+                                                 fleet-edge \
+                                                 fleet-api \
+                                                 service-catalog \
+                                                 trace-edge
+          ```
+        - [ ] Check the deployment is successful on the [Go tracer dashboard]
+    - [ ] Deploy the reliability-environment services:
+        - [ ] Run [the gitlab pipeline](https://gitlab.ddbuild.io/DataDog/datadog-reliability-env/-/pipelines/new)
+          with the variable `DD_TRACE_GO_CANDIDATE_VERSION` set to the release candidate version you want to deploy.
+        - [ ] Check the deployment is successful on the [Go rel-env dashboard].
+    - [ ] Update every dd-trace-go team on `#guild-dd-go` so that they can perform any further checks.
+
+- Let the release candidate run on staging and rel-env for at least 24 hours and review all the dashboards, especially:
+    - [ ] Check the memory usage trend of rel-env which should be flat over time on the [Go rel-env dashboard].
+    - [ ] Check de deployment tab of some deployed staging services to see if there was any negative impact on the
+      service latency and memory usage (
+      eg. https://ddstaging.datadoghq.com/apm/services/trace-edge/operations/http.request/deployments).
+    - [ ] Check through the [Go tracer dashboard] to look for any anomalies such as high memory or cpu use, increased
+      number of dropped traces, increase in tracer errors.
+
+- If anything went wrong during this release candidate, you can fix the problems on this release branch and go through a
```

Comment:
Maybe be a little more explicit about what they should do here. It sounds like the suggestion is to forward fix on the same branch, and tag another release, repeating all the steps above. Is that correct?

Reply:
Yes this is it. I tried to rephrase and clarify this sentence.